app/client/packages/design-system/widgets/src/components/Sheet/stories/SimpleSheet.tsx

```
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import { Button } from "../../Button";
+import { Sheet } from "../index";
+import type { SheetProps } from "../src/types";
+
+export const SimpleSheet = (props: Omit<SheetProps, "children">) => {
+  const [isOpen, setIsOpen] = useState(props.isOpen);
+
+  return (
+    <>
+      <Button onPress={() => setIsOpen(!Boolean(isOpen))}>Sheet trigger</Button>
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)